frontend/ui/src/components/custom-sheet.tsx

```
@@ -0,0 +1,64 @@
+  children,
+}) => {
+  return (
+    <AnimatePresence>
+      {open && (
+        <dialog
+          className={cn(
+            kanbanFilter
+              ? "relative z-50 2xl:w-[40rem] w-[30rem] h-full flex-shrink-0"
+              : "fixed top-0 left-0 h-screen w-screen z-50",
+          )}
+          role="dialog"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<dialog>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)